x-pack/plugins/security_solution/public/management/components/console/components/console_state/state_update_handlers/handle_execute_command.test.tsx

```diff
@@ -241,6 +241,25 @@ describe('When a Console command is entered by the user', () => {
     });
   });
 
+  it('should show error for --help if command definition `validate()` callback return a message', async () => {
```

Comment:
nit: typo `...returns a message`

Reply:
Copy pasting at its best :). To make it up I went through the whole file and fixed typos :).